mods/ts/rules/nod-vehicles.yaml

```diff
@@ -138,7 +140,7 @@ TTNK:
 		RequiresCondition: !undeployed && real-actor
 	AttackFrontal:
 		Voice: Attack
-		RequiresCondition: undeployed
+		RequiresCondition: !empdisable && undeployed
```

Comment:
Shouldn't these be 
```
RequiresCondition: undeployed
PauseOnCondition: empdisable
```
and
```
RequiresCondition: deployed
PauseOnCondition: empdisable
```
?  We do want to switch these off completely when in the opposite deploy state.